src/backend/wsl.ts

```diff
@@ -1119,6 +1119,18 @@ export default class WSLBackend extends events.EventEmitter implements K8s.Kuber
     await this.progressTracker.action(description, 10, async() => {
       try {
         this.setState(K8s.State.STARTING);
+
+        // Register a new tunnel for RD Guest Agent
+        if (process.platform === 'win32') {
+          this.vtun.addTunnel({
+            name:                  'Rancher Desktop Guest Agent',
+            handshakePort:         17382,
+            vsockHostPort:         17381,
+            peerAddress:           '127.0.0.1',
+            peerPort:              3040,
+            upstreamServerAddress: 'npipe:////./pipe/privilegedservice',
```

Comment:
This is not the correct address for the guest agent; which one is it?

Reply:
This is the address for privileged service (upstream), the RD guest agent address is over TCP `127.0.0.1:3040`